sdk/python/endpoints/batch/heart-classifier-mlflow/code/batch_driver_parquet.py

```diff
@@ -0,0 +1,30 @@
+import os
+import mlflow
+import pandas as pd
+from pathlib import Path
+
+
+def init():
+    global model
+    global output_path
+
+    # AZUREML_MODEL_DIR is an environment variable created during deployment
+    # It is the path to the model folder
+    # Please provide your model's folder name if there's one:
+    model_path = os.path.join(os.environ["AZUREML_MODEL_DIR"], "model")
+    output_path = os.environ["AZUREML_BI_OUTPUT_PATH"]
+    model = mlflow.pyfunc.load_model(model_path)
+
+
+def run(mini_batch):
+    for file_path in mini_batch:
+        data = pd.read_csv(file_path)
```

Comment:
We are reading a CSV here but the file name mentions parquet which is not yet supported in batch endpoints. Should we update the name?

Reply:
The script reads CSV files but outputs parquet files. The name is because of the output file type. Maybe I can reframe this example to read and write both parquet so it is less confusing. Thanks for the idea!